docs/quickstart.md

```diff
@@ -1,3 +1,4 @@
+Ulanzi device is switched on/off by holding left and right buttons together.
```

Comment:
Thats a hardware specific feature and has nothing to do with AWTIRX Light software. After unboxing you have to turn it on anyway so the user knows it before he flash awtrix light

Reply:
I thought giving complete info for the supported hardware would be nice.
I've removed it.